src/main/java/org/jenkinsci/plugins/workinghours/WorkingHoursConfig.java

```diff
@@ -131,4 +136,24 @@ public void setExcludedDates(
         this.excludedDates = value;
         save();
     }
+
+    /**
+     * Gets the message to be displayed when the job is blocked
+     *
+     * @return default message if the value is null
```

Comment:
nit: it really returns the custom message, or the default message if the custom message is null